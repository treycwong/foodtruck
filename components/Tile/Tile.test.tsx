import { render } from "@/test-utils";

import { Tile } from "./Tile";

describe("Tile test cases", () => {
  it("Tile render check", () => {
    const { asFragment } = render(
      <Tile header="Lorem ipsum dolor sit amet">
        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quia maxime
        officia sed magnam nisi dignissimos vitae temporibus omnis dolore
        nostrum.
      </Tile>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
