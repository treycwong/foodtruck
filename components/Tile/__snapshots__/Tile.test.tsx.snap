// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile test cases Tile render check 1`] = `
<DocumentFragment>
  <section
    class="css-brv202"
  >
    <h2>
      Lorem ipsum dolor sit amet
    </h2>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quia maxime officia sed magnam nisi dignissimos vitae temporibus omnis dolore nostrum.
  </section>
</DocumentFragment>
`;
